ui/app/models/ssh/ca-config.js

```diff
@@ -38,16 +38,15 @@ export default class SshCaConfig extends Model {
   @attr('string') backend; // dynamic path of secret -- set on response from value passed to queryRecord
   @attr('string', { sensitive: true }) privateKey; // obfuscated, never returned by API
   @attr('string') publicKey;
```

Comment:
I know I said I didn't touch SSH, but I did clean up the model to match the pattern of the others.